Source/JavaScriptCore/builtins/IteratorHelpers.js

```diff
@@ -54,10 +54,10 @@ function performIteration(iterable)
 }
 
 @linkTimeConstant
-function wrappedIterator(iterator)
+function wrapIterator(iterator)
 {
     var wrapper = @Object.@create(null);
-    wrapper.@@iterator = function() { return iterator; }
+    wrapper.@@iterator = function() { return iterator; };
     return wrapper;
```

Comment:
We should use normal object literal here since it is more efficient than `Object.create(null)`.
Also, I wonder if whether we can make this faster by doing,
```
return {
    iterator,
    @@iterator: function() { return this.iterator; },
};
```
so that we can avoid creation of environment.